pftp/socket_control.go

```diff
@@ -0,0 +1,33 @@
+package pftp
+
+import (
+	"net"
+	"syscall"
+
+	"golang.org/x/sys/unix"
+)
+
+// send EOF to write
+func sendEOF(conn net.Conn) {
```

Comment:
Do you need a return value for this method? Are you sure you will succeed?

Reply:
I thought just use like (net.Conn).Close()
But, it contains error type return too.
I'll fix it. thanks!

Reply:
🆗

Reply:
I added return value check.

If send EOF failed, close connection.

https://github.com/pyama86/pftp/pull/32/commits/5123888b7996686f81fad407027e13b5d1db8fb3